conf/provisioning/alerting/sample.yaml

```diff
@@ -0,0 +1,165 @@
+#   - orgId: 123
+#     # <string, required> name of the rule group
+#     name: my_rule_group
+#     # <string, required> name of the folder the rule group will be stored in
+#     folder: my_first_folder
+#     # <duration, required> interval that the rule group should evaluated at 
+#     interval: 60s
+#     # <list, required> list of rules that are part of the rule group  
+#     rules:
+#       # <string, required> unique identifier for the rule
+#     - uid: my_id_1
+#       # <string, required> title of the rule that will be displayed in the UI
+#       title: my_first_rule
+#       # <string, required> query used for the condition
+#       condition: A
+#       # <list, required> list of query objects that should be executed on each
+#       #                  evaluation - should be obtained via the API
+#       data:
+#       - ...
+#       # <string> UID of a dashboard that the alert rule should be linked to
+#       dashboardUid: my_dashboard
+#       # <int> ID of the panel that the alert rule should be linked to
+#       panelId: 123
+#       # <string> state of the alert rule when no data is returned
+#       #          possible values: "NoData", "Alerting", "OK", default = NoData
+#       noDataState: Alerting
+#       # <string> state of the alert rule when the query execution 
+#       #          fails - possible values: "Error", "Alerting", "OK"
+#       #          default = Alerting
+#       # <duration, required> how long the alert should fire before alerting
+#       for: 60s
+#       # <map<string, string>> map of strings to pass around any data 
+#       annotations:
+#         some_key: some_value
+#       # <map<string, string> map of strings to filter and 
+#       #                      route alerts
+#       labels:
+#         team: sre_team_1
+
+# # List of alert rule UIDs that should be deleted
+# deleteRules:
+#    # <int> organization id, default = 1
+#  - orgId: 1
+#    # <string, required> unique identifier for the rule
+#    uid: my_id_1
+
+# # List of contact points to import or update
+# contactPoints:
+#     # <int> organization id, default = 1
+#   - orgId: 1
+#     # <string, required> name of the contact point
+#     name: cp_1
+#     receivers:
+#       # <string, required> unique identifier for the receiver
+#     - uid: first_uid
+#       # <string, required> type of the receiver
+#       type: prometheus-alertmanager
+#       # <object, required> settings for the specific receiver type
+#       settings:
+#         url: http://test:9000
+
+# # List of receivers that should be deleted
+# deleteContactPoints:
+# - orgId: 1
+#   uid: first_uid
+
+# # List of notification policies to import or update
+# policies:
+#     # <int> organization id, default = 1
+#   - orgId: 1
+#     # <string> name of the receiver that should be used for this route
+#     receiver: grafana-default-email
+#     # <list> The labels by which incoming alerts are grouped together. For example,
```

Comment:
```suggestion
#     # <list<string>> The labels by which incoming alerts are grouped together. For example,
```
A few other places too. Spots where we have a list of objects is fine to leave is `list` as the element type is obvious. But, cases like this where a list is the leaf level type, we should say what the element type is just like we did for maps.